library/Icingadb/Widget/ObjectsStateBadges.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/* Icinga DB Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Module\Icingadb\Common\StateBadges;
+
+/**
+ * State badges for the objects
```

Comment:
I know I've requested to rename this class. But now I wonder what *the objects* are actually?

I believe you don't want to hear this, but what about `DependencyNodeStateBadges`? :rofl: 

It was about to generalize the name, so *members*, *parents* and *children* were all too specific. Though, *objects* is way to arbitrary now. *dependency node* on the other hand fits all three specific names and is still general enough. :sweat_smile: